lib/screen/onboarding/view_address/view_existing_address.dart

```diff
@@ -32,7 +34,10 @@ class ViewExistingAddress extends StatefulWidget {
 class _ViewExistingAddressState extends State<ViewExistingAddress> {
   final _controller = TextEditingController();
   bool _isError = false;
-  bool _isProcessing = false;
+  String _address = '';
+  bool _isValid = false;
```

Comment:
looks like we're having many state & more logic now, can we refactor to using bloc for this page?

Reply:
can we do that later?

Reply:
I think it's okay if you have other important tasks